extensions/data-workspace/src/test/dialogs/newProjectDialog.test.ts

```diff
@@ -48,25 +53,27 @@ suite('New Project Dialog', function (): void {
 		dialog.model.name = `TestProject_${new Date().getTime()}`;
 		dialog.model.location = os.tmpdir();
 		dialog.workspaceInputBox!.value = 'test';
-		should.equal(await dialog.validate(), false, 'Validation should fail because workspace does not end in .code-workspace');
+		should.equal(await dialog.validateNewWorkspace(false), false, 'Validation should fail because workspace does not end in .code-workspace');
```

Comment:
Why are you doing this instead of calling validate and then stubbing out whatever else validate is calling? That would make the most sense - by calling this function directly you're bypassing some of the normal logic which doesn't seem right

Reply:
there was a separate test for the other validation in validate(), but I switched this back to to call validate instead of the new workspace validation